components/list/list.js

```diff
@@ -342,6 +343,20 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 		if (focusable) focusable.focus();
 	}
 
+	_handleListItemMenuItemClick(e) {
```

Comment:
Note that this should actually just be when `current` changes so I might want to use the property-change event instead.

This was the piece that I had the most challenge with as I wasn't sure if I should be somehow using the list subscriber pattern.

Problem: when a non-currently selected item becomes current item we want any previously current item to no longer be, and any parent items of that item to no longer have `aria-current="location"`.

This generally works but I'm wondering if there's a more list-pattern-friendly way.